boilerplate/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM node:8.9-alpine
+
+# Initialize app directory
+ENV APP_DIR "/usr/app"
+ENV PYTHONPATH="/:${PYTHONPATH}"
+
+RUN mkdir -p $APP_DIR
+WORKDIR $APP_DIR
+
+#PM2 will be used as PID 1 process
+RUN npm install -g pm2@1.1.3
```

Comment:
Have to tested pm2's behavior? All cases - happy case, application dies, ....

Reply:
yes...but can you point me to a resource which had absolute ways to test this